dandi/bids_validator_xs.py

```diff
@@ -711,6 +711,16 @@ def validate_bids(
         regex_schema,
         debug=debug,
     )
+    # Record schema version.
+    # Not sure whether to incorporate in validation_result.
+    if bids_schema_dir == os.path.join(
+        os.path.abspath(os.path.dirname(__file__)),
+        "data/schema",
```

Comment:
Will this work on Windows?  It would be better to instead do `"data", "schema"` — or, even better, use `pathlib` instead.